src/axom/quest/util/mesh_helpers.cpp

```diff
@@ -109,9 +111,9 @@ axom::sidre::Group* make_structured_blueprint_box_mesh_3d(axom::sidre::Group* me
                 "Cannot allocate coordset '" + coordsetName + "' in blueprint mesh '" +
                   meshGrp->getName() + "'.  It already exists.");
 
-  topoGrp->createView("type")->setString("structured");
-  topoGrp->createView("coordset")->setString(coordsetName);
-  auto* dimsGrp = topoGrp->createGroup("elements/dims");
+  topoGrp->createView("type")->setString("structured", hostAllocId);
```

Comment:
I thought you mentioned strings and scalars were always host allocated?  Or is that just scalars that are always host?

Reply:
That's the current behavior, which is useful, but unintended.  Our consensus was for everything to default to the group's allocator unless overridden, as I do on line 114.

Reply:
Important note: I have gone with a tuple allocator (and the notion of a tuple, which is like an array but is treated like a scalar for memory and handling purposes).  Using a single allocator with overrides for specific Views was horribly tedious and ultimately unworkable.